btf/btf.go

```diff
@@ -819,8 +819,42 @@ var haveMapBTF = internal.FeatureTest("Map BTF (Var/Datasec)", "5.2", func() err
 	return nil
 })
 
+// haveProgBTF attempts to load a BTF blob containing a Func and FuncProto. It
+// is used as a proxy for ext_info (func_info) support, which depends on
+// Func(Proto) by definition.
+var haveProgBTF = internal.FeatureTest("Program BTF (func/line_info)", "5.0", func() error {
```

Comment:
If we keep `haveBTF` this function could / should short circuit that?

Reply:
Ack, makes sense, though doesn't really matter in the grand scheme of things since the results are all cached.

Reply:
Done.